### DISCUSSION TITLE
Please help I'm having trouble cut and pasting visualizations  Into a Playlist

### DISCUSSION THREAD

**Discussion by howthefdoicreateaplaylist** [projectM-visualizer/projectm#580] [General]
I'm trying to follow the directions it tells me that I can cut a visualization And paste it into my own Playlist However After I create my Playlist and cut my visualization My Playlist Please help thank you

**Comment by kblaschke** [comment]
Please tell us which frontend and on which operating system you're using projectM and which directions you actually refer to.

There are many different apps and plug-ins that use projectM, so without knowing any details, we're not able to answer this question.
